libr/core/cmd_search.c

```diff
@@ -1188,14 +1175,8 @@ static void print_rop(RCore *core, RList *hitlist, char mode, bool *json_first)
 
 	switch (mode) {
 	case 'j':
-		//TODO PJ
-		// Handle comma between gadgets
-		if (*json_first) {
-			*json_first = 0;
-		} else {
-			r_cons_strcat (",");
-		}
-		r_cons_printf ("{\"opcodes\":[");
+		pj_o (pj);
+		pj_ka (pj, "opcodes");
 		r_list_foreach (hitlist, iter, hit) {
 			ut8 *buf = malloc (hit->len);
```

Comment:
check for null

Reply:
> check for null

Would it be nice to have some malloc/calloc/realloc wrapper? In C++ if a `new` fails, the program automatically terminates and nothing needs to be done because once you use up heap memory, you actually can do nothing to recover.